src/sentry/sentry_metrics/consumers/indexer/batch.py

```diff
@@ -71,16 +72,24 @@ def invalid_metric_tags(tags: Mapping[str, str]) -> Sequence[str]:
     return invalid_strs
 
 
+def extract_use_case_id(mri: str) -> Optional[str]:
+    use_case_mapping = {"transactions": "performance", "sessions": "release-health"}
```

Comment:
This mapping is good for the purposes of demonstrating this PR. 

Just as a heads up, in my other PR (where we're defining `UseCaseID`), I'll introduce a Namespace : UseCaseID mapping, so maybe we can just use that here ultimately.